### PR TITLE
Added Favicon and Sidebar

### PR DESCRIPTION
**Pull request by hussiantabha**
Now with the current video users can see video suggested to them

### CODE REVIEW

File: src/components/DisplaySidebarVideo.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+            avatar_url,
+            creator,
+            views,
+            uploaded,
+            duration,
+          }) => {
+            return (
+              <div key={_id} className="siderbar-display-video-container">
+                <div>
+                  <img src={thumbnail} className="sidebar-display-container" />
```
Comment (shivanigangadharan) [inline src/components/DisplaySidebarVideo.js]: can add `alt` here

File: src/css/displayvideo.css [hunk 2]
```diff
@@ -42,7 +48,7 @@
 }
 .sidebar-video-img {
   width: 100%;
-}
+} */
```
Comment (shivanigangadharan) [inline src/css/displayvideo.css:58]: remove comments if not reqd

**Review by shivanigangadharan — APPROVED** [review]
Code looks good, nice work on the UI as well.

File: src/css/displayvideo.css [hunk 3]
```diff
@@ -109,6 +115,34 @@
...
+  justify-content: space-around;
+}
+.sidebar-displayVideo {
+  display: flex;
+  flex-flow: column wrap;
+  row-gap: 1rem;
+}
+.siderbar-display-video-container {
+  display: flex;
+  width: 520px;
```
Comment (Kothalkarpriya) [inline src/css/displayvideo.css:143]: for more than 10px you can use rem

**Review by Kothalkarpriya — APPROVED** [review]
The code is clean and good. You can merge

**Pull request merged**